### DISCUSSION TITLE
HSR cannot talk by tts module

### DISCUSSION THREAD

**Discussion by LimSeunghyeon1** [HumanSupportRobot/QandA#45] [Q&A]
Hello, we have a problem that the robot doesn't talk by tts module.
We tried publishing a topic "/talk_request".
We checked a diagnosis at hsrb.local/admin, and a speaker works fine. We think a speaker(hardware) isnt a problem, rather it is a matter of ros communication or other software problems.
Could you give us a useful advice?

Thank you.

**Comment by hsr-admin** [comment]
Since the diagnosis speaker test that plays the wav file is working fine, can you please confirm the following cause that I can think of.

- The following license file is not in the robot

    ` /opt/tmc/vt/**/M16/data-common/verify/verification.txt `

- ROS_IP is not set

- The interface of the MAC address written in hostid in the license file is not up.

**Reply by YuqianJiang** [reply]
Hello, we are experiencing the same issue. The HSR still makes the sound when it boots up, so the speaker is not broken. But it does not say "HSR Start" or speak with /talk_request.

**Reply by LimSeunghyeon1** [reply]
We solved the problem by the clean install.
